web/src/BeaconPanel.jsx

```
@@ -0,0 +1,361 @@
+          title="Send (Ctrl+Enter)"
+          aria-busy={isSending}
+        >
+          {isSending ? (
+            <>
+              <Spinner />
+              Sending...
+            </>
+          ) : "Send"}
+        </button>
+      </div>
+      {status && <div className="compose-status" role="status" aria-live="polite">{status}</div>}
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)